src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -21,68 +57,74 @@
              [ngbPopover]="healthChecks"
              popoverClass="info-card-popover-cluster-status">
-             {{ healthData.health.status | healthLabel | uppercase }}
+            {{ healthData.health.status | healthLabel | uppercase }}
           <i *ngIf="healthData.health?.status !== 'HEALTH_OK'"
              class="fa fa-exclamation-triangle"></i>
         </div>
       </div>
-      <div class="col-lg-3 h-50 d-flex flex-column border justify-content-center align-items-center">
-        <span class="bold">Upgrade Status</span>
-        <ng-container *ngIf="upgradeInfo.versions.length > 0; else noUpgradesAvailable">
-          <span class="mt-2"
-                i18n>
-          <i [ngClass]="[icons.up]"
-             class="text-info"></i>
-            Upgrade available</span>
-          <div i18n-ngbTooltip
-               [ngbTooltip]="(healthData.mgr_map | mgrSummary).total <= 1 ? 'To upgrade, you need minimum 2 mgr daemons.' : ''">
-            <button class="btn btn-accent mt-2"
-                    id="upgrade"
-                    aria-label="Upgrade now"
-                    [disabled]="(healthData.mgr_map | mgrSummary).total <= 1"
-                    (click)="startUpgradeModal()"
-                    i18n>Upgrade now</button>
-          </div>
-        </ng-container>
-      </div>
-      <div class="col-lg-3 h-50 d-flex flex-column border justify-content-center align-items-center">
-        <span class="bold">MGR Count</span>
-        <span class="mt-1">
+    </cd-card>
+
+    <cd-card class="col-sm-3 px-3 d-flex"
+             cardTitle="MGR Count"
+             i18n-cardTitle
+             aria-label="MGR Count"
+             i18n-aria-label
+             id="mgrCount">
+      <div class="d-flex flex-column justify-content-center align-items-center">
+        <h5>
           <i class="text-success"
              [ngClass]="[icons.success]"
              *ngIf="(healthData.mgr_map | mgrSummary).total > 1; else warningIcon">
           </i>
           {{ (healthData.mgr_map | mgrSummary).total }}
-        </span>
+        </h5>
       </div>
+    </cd-card>
 
-      <div class="d-flex mt-3 p-0">
-        <dl class="w-50"
-            *ngIf="fsid$ | async as fsid">
-          <dt class="bold mt-5"
-              i18n>Cluster FSID</dt>
-          <dd class="mt-2">{{ fsid }}</dd>
+    <div class="d-flex mt-3">
+      <dl class="w-50"
+          *ngIf="fsid$ | async as fsid">
+        <dt class="bold mt-5"
+            i18n>Cluster FSID</dt>
+        <dd class="mt-2">{{ fsid }}</dd>
+
+        <ng-container *ngIf="info$ | async as info; else loadingDetails">
           <dt class="bold mt-5"
               i18n>Release Image</dt>
-          <dd class="mt-2">{{ upgradeInfo.image }}</dd>
+          <dd class="mt-2">{{ info.image }}</dd>
           <dt class="bold mt-5"
               i18n>Registry</dt>
-          <dd class="mt-2">{{ upgradeInfo.registry }}</dd>
-        </dl>
+          <dd class="mt-2">{{ info.registry }}</dd>
```

Comment:
If these are URLs, I'd make those a hyperlink, or would add a copy-to-clipboard widget. 

And if image already contains the registry part, we can simply remove one of them to deduplicate text (BTW, card body text should be smaller than the card header one).

![image](https://github.com/ceph/ceph/assets/37327689/86c8db1b-d9b1-4b94-afef-1bfea25c7d78)

Reply:
I'll keep this in mind for the follow-up PR. Thanks